src/components/FAB/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import FAB from './FAB';
+import FabPropTypes from './FabPropTypes';
+
+function Fab({onPress, isActive}) {
+    return <FAB onPress={onPress} isActive={isActive} />;
+}
```

Comment:
I'm not following why we can't use the `KeyboardAvoidingView` everywhere? What is the problem exactly?

Reply:
1. For android we have no need to use `KeyboardAvoidingView`
2. If we want to add `KeyboardAvoidingView` for android then we need to make many style changes(for both big and small screen) on the `SidebarScreen` because  `FAB` onPress not working with `KeyboardAvoidingView` in android.

Reply:
Alright, I'll take your word for it. Although I'm not sure which style changes you mean in this case.

Reply:
We should add some kind of comment so that we will understand what purpose splitting these files in two serves?